src/main/java/be/seeseemelk/mockbukkit/ServerMock.java

```diff
@@ -157,6 +160,14 @@ public ServerMock()
 		logger.setLevel(Level.ALL);
 	}
 
+	public ServerSettingsMock getServerSettings(){
```

Comment:
Please follow MockBukkit's bracket style.
It's also a good practice to add nullabity annotations (NotNull / Nullable) where possible to indicate whether a method can or cannot return null values.

```suggestion
	public @NotNull ServerSettingsMock getServerSettings()
	{
```